engine/src/main/java/org/terasology/engine/module/ModuleManagerImpl.java

```diff
@@ -83,6 +85,11 @@ public ModuleManagerImpl() {
 
         setupSandbox();
         loadEnvironment(Sets.newHashSet(engineModule), true);
+        installManager = new ModuleInstallManager(this, masterServerAddress);
+    }
+
+    public ModuleManagerImpl(Config config) {
+        this(config.getNetwork().getMasterServer());
```

Comment:
In my understanding it could be changed without breaking config compatibility by replacing
`public String getMasterServer() { return masterServer; }`
with
`public String getMetaServer() { return masterServer; }`
 in `NetworkConfig` class, i.e. renaming the getter but not the field. Not sure if it's good practice though.

Or alternatively we could keep both fields (`masterServer` and `metaServer`), then in the getter returning `metaServer` by default but falling back to `masterServer` if null, and copying the value from the old field to the new one when the config is saved. This way we could keep compatibility with old config files for a certain amount of time, then one day in the future "deprecating" the old field name.